queries/leo/highlights.scm

```diff
@@ -168,10 +172,12 @@
 
 [ 
   (address_literal)
-  (affine_group_literal) 
+  ((affine_group_literal) (#set! "priority" 101))
```

Comment:
Why priority for this?

Can't it be moved downwards instead

Reply:
Because an example affine_group_literal looks like this: `(1,2)group` unfortunately it has both parentheses and comma. While we want affine_group_literal to be colored evenly,if it is not prioritized, then the first paren and the comma will have different color - no good. After prioritization it all looks as intended: nice and even.

Reply:
BTW your speed is amazing. Just made the PR, and you commented... Wow... :)

Reply:
are you sure you want it to be colored evenly? it looks like some sort of point type/value

Reply:
Yes. This should be colored like a number literal. Evenly. This value represents a single point on a Twisted Edwards curve. It has addition and subtraction defined. So it behaves like number. So YES.

Reply:
I know it is weird, but the whole construct is weird.

Reply:
What about capturing everything below it?
Can something like `(affine_group_literal _ @capture_name)` work?

Reply:
Let me give you a real example: the literal `2group` is equal to `(2, 5553594316923449299484601589326170487897520766531075014687114064346375156608)group`. They are just a different notation of the same literal. The second one is NOT an expression, it is a literal. This is what is emphasized with even coloring. If we would allow different colors for the parens and comma, we would give the impression that user has entered some kind of expression. With even colors it is immediately obvious that it is just a group literal. 
The coordinates are numbers without type. But in leo there no such thing as `2` only `2u32` or the likes. We shouold not color them the same as number literals. 
We are discussing a thing that is not important. There is no programmer in the world, that will use the second notation instead of the first. It is just nice that we are prepared for this. I know its hard to to get the feeling of a language that has `(x,z)group` literals, where `then` is not a conditional, and has four different kinds of function-like constructs. If you insist I can do sub coloring of course, but that is against the languages logic.